src/handlers/localstorage/globalSettings.js

```diff
@@ -63,3 +64,20 @@ export const saveTransactionSignatures = transactionSignatures =>
  */
 export const getTransactionSignatures = () =>
   getGlobal(TRANSACTION_SIGNATURES, {});
+
+/**
+ * @desc get dev enabled preference
+ * @param  {Boolean}  [value]
+ */
+export const getTestnetsEnabled = async preference => {
```

Comment:
`preference` should not be passed in here

Reply:
oops yeah i did a refactor/name change and missed this